reference-project/src/main/java/com/devonfw/demoquarkus/rest/v1/controller/SecureRestController.java

```diff
@@ -0,0 +1,85 @@
+package com.devonfw.demoquarkus.rest.v1.controller;
+
+import org.eclipse.microprofile.jwt.JsonWebToken;
+import javax.annotation.security.PermitAll;
+import javax.annotation.security.RolesAllowed;
+import javax.enterprise.context.RequestScoped;
+import javax.inject.Inject;
+import javax.ws.rs.*;
+import javax.ws.rs.core.*;
+
+@RequestScoped
+@Path("/secure")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public class SecureRestController {
+    
+    @Inject
+    JsonWebToken jwt;
```

Comment:
I am not familiar with this style so sorry for asking:
You are making the REST service `@RequestScope` so physically for each request an own instance is created and the JWT passed as HTTP parameter is injected into this stateful bean instead of passing it as method parameter?

I rahter find this a magical confusion but maybe this is quarkus best practice.
However, I do not get why then we receive `SecurityContext` as paramter than and the JWT as injection.